cmd/logs.go

```diff
@@ -72,6 +72,9 @@ type logsOptions struct {
 	size       int64
 }
 
+var logsSingleFilters = []string{"compute_backend", "docker_img", "status"}
+var logsMultiFilters = []string{"step"}
```

Comment:
The fact that some `logs` command filters are single-natured and some multi-natured is a bit hidden here.  Perhaps we can move this REANA system knowledge outside into some "configuration" file?  See also #80. Something for later?

Reply:
I agree, it'd be useful to move some constants like these to a config file, it probably makes sense to do it in #80 

The single or multi nature of the filters also feel a bit weird to me, but that comes from the Python Client. In this example, why do we not let the users choose multiple status or compute_backend? This doesn't seem to be in the help message either

Reply:
Yeah, it's not currently consistent in the Python client.  For example:

- `reana-client logs -w root6-roofit-yadage-kubernetes --filter status=pending --filter status=failed`
- `reana-client logs -w root6-roofit-yadage-kubernetes --filter step=gendata --filter step=fitdata`

The original thinking behind filtering options was the following: you get results, and you filter what you want to narrow to, and _then_ you filer on the filtered results what you want to narrow ever more. In this optics, both of the above commands should return nothing.  But, currently, whilst the first command conforms to this logic, the latter command returns both steps instead of nothing.  So the analogy with Unix pipe filters is retained for the "status" filter, but broken for the "step" filter, one could say.

P.S. The obvious usage would be something like `--filter name=roofit --filter status=finished --filter step=gendata`. But the above commands help to think about joining up filters more easily.

We can ticketise this for later, and either improve the documentation, or change the behaviour.

Reply:
Sounds good, I think we can merge this then